.gitignore

```
@@ -36,7 +36,7 @@ bld/
 vs
 
 # Visual Studio Code cache/options directory
-.vscode/
+.vscode
```

```suggestion
.vscode/
```
You probably want to keep the slash here since its a dir.